addons/source-python/plugins/wcs/core/modules/races/manager.py

```diff
@@ -138,6 +138,12 @@ def usable_by(self, wcsplayer):
             if team >= 2 and not restrictteam == team:
                 return RaceReason.TEAM
 
+        restrictbot = self.config.get('restrictbot')
+
+        if restrictbot:
+            if wcsplayer._is_bot and restrictbot == 1:
```

Comment:
As you're already checking if `restrictbot` is true in the prior line, you don't have to check if it's equal 1 here (if it's anything other than `0`, they probably want to enable it).